spec/boolean_spec.rb

```diff
@@ -255,17 +256,21 @@
 
     it 'lesbian OR gay videos', jira: ['VUF-300', 'VUF-301', 'VUF-311'] do
       resp = solr_resp_doc_ids_only('q' => 'lesbian OR gay', 'fq' => 'format:Video')
-      expect(resp.size).to be >= 1520
-      expect(resp.size).to be <= 1770
+      expect(resp.size).to be >= 1800
+      expect(resp.size).to be <= 1900
     end
 
-    context 'street art and graffiti', jira: 'VUF-1013' do
+    context 'street art and graffiti', jira: 'VUF-1013', fixme: true do
       it '((street art) OR graffiti) AND aspects' do
+        # this search returns 47 results when the solr query is sent from SearchWorks
+        # "q"=>"( ( _query_:\"{!edismax qf=$qf_subject pf=$pf_subject pf3=$pf3_subject pf2=$pf2_subject}street art\" OR _query_:\"{!dismax qf=$qf_subject pf=$pf_subject pf3=$pf3_subject pf2=$pf2_subject}graffiti\" ) AND _query_:\"{!dismax qf=$qf_subject pf=$pf_subject pf3=$pf3_subject pf2=$pf2_subject}aspects\" )"
```

Comment:
perhaps we can work together to change this to be the query actually sent by solr.  I'm thinking of spinning up SW locally, pointing to a particular Solr instance, and then tailing the logs to see what the query sent was ...